libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/metrics/SessionState.kt

```diff
@@ -0,0 +1,83 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation.metrics
+
+import android.location.Location
+import android.text.TextUtils
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.api.directions.v5.models.RouteLeg
+import com.mapbox.core.constants.Constants
+import com.mapbox.geojson.utils.PolylineUtils
+import com.mapbox.services.android.navigation.v5.internal.navigation.routeprogress.MetricsRouteProgress
+import com.mapbox.services.android.navigation.v5.utils.extensions.ifNonNull
+import java.util.Date
+
+internal data class SessionState @JvmOverloads constructor(
+    var secondsSinceLastReroute: Int = -1,
```

Comment:
should be this fields `val`?

Reply:
No, cause of `NavigationTelemetry#line416`